scripts/ansible/roles/public-ami/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+ava_user: chain4travel
+ava_group: admin
+network: columbus
```

Comment:
Even we don't run the ansible thingy, I would argue that kopernikus is better than columbus here

Reply:
That was moved from camino-node as it is. Should we change this in this PR? There are a lot of mistakes in those thingies, I didn't want to touch them at all in this PR, cause, imo, its better to address them systematically in dedicated PR later.

Reply:
There will not be too many of them, and because the files are all "new files" in this PR I suggest to do it now.

Reply:
ok, then I'll also do changes from @knikos comments